experiments/AMIP/modular/components/land/bucket_init.jl

```diff
@@ -190,10 +190,10 @@ function bucket_init(
     if albedo_type == "map_static" # Read in albedo from static data file (default type)
         # By default, this uses a file containing bareground albedo without a time component. Snow albedo is specified separately.
         if ClimaComms.iamroot(comms_ctx)
-            albedo = BulkAlbedoStatic{FT}(regrid_dirpath, α_snow = α_snow, comms = comms_ctx)
+            albedo = BulkAlbedoStatic{FT}(regrid_dirpath, α_snow = α_snow)
```

Comment:
I suppose this is a consequence of ClimaLSM updates ?

Reply:
Yes! ClimaLSM should have had a minor release bc of this interface change